policy-report/samples/sample-cis-k8s.yaml

```diff
@@ -12,28 +12,15 @@ summary:
   pass: 8
   fail: 2
   warn: 0
-  info: 0
   error: 0
   skip: 0
 results:
   - policy: api-server:anonymous-auth
     message: ensure that --anonymous-auth argument is set to false
-    status: warn
     scored: true
```

Comment:
Why did we remove the `status`? I believe that is a required field right?

Reply:
I guess if we keep the `status` as a field in the results section, we are getting the following error:
```error: error validating "samples/sample-cis-k8s.yaml": error validating data: ValidationError(PolicyReport.results[0]): unknown field "status" in io.wgpolicyk8s.v1alpha1.PolicyReport.results; if you choose to ignore these errors, turn validation off with --validate=false``` 

Besides that, I was not able to find `status` in https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/api/v1alpha1/policyreport_types.go, so please tell what can we figure out from these and what should we do?

Reply:
Thanks @JimBugwadia , changed `status` to `result` and now the above errors have gone. Thanks a lot!